aspnetcore/fundamentals/logging/index.md

```diff
@@ -796,15 +796,76 @@ On Linux, this provider writes logs to */var/log/message*.
 logging.AddDebug();
 ```
 
-### EventSource provider
+### Event Source provider
 
-For apps that target ASP.NET Core 1.1.0 or later, the [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing. On Windows, it uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803). The provider is cross-platform, but there are no event collection and display tools yet for Linux or macOS.
+The [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing cross-platform. On Windows, the provider uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803).
```

Comment:
> The [Microsoft.Extensions.Logging.EventSource](https://www.nuget.org/packages/Microsoft.Extensions.Logging.EventSource) provider package can implement event tracing cross-platform. On Windows, the provider uses [ETW](https://msdn.microsoft.com/library/windows/desktop/bb968803).


I would say it writes to an EventSource with the name "Microsoft-Extensions-Logging", then point to the event source article (@noahfalk we need the EventSource docs to talk about the cross platform-ness)

Reply:
> point to the event source article

Link? ... if you're referring to the one that I have there, try the next commit for an update to this paragraph. :point_down: